server/src/controllers/proposals.ts

```diff
@@ -56,8 +60,28 @@ const acceptProposal = async (req: Request, res: Response) => {
     await Proposal.destroy({ where: { isAccepted: false, jobId }, transaction: t });
     await job?.update({ isOccupied: true }, { transaction: t });
   });
+  const userInfo: any = await Freelancer.findOne({
```

Comment:
```js
  interface FreelancerInfo extends FreelancerInstance {
    user?: { name: string, email: string }
  }
  const userInfo: FreelancerInfo = await Freelancer.findOne({
```
mybe we can use this instead of any